google-cloud-spanner/src/main/java/com/google/cloud/spanner/SpannerOptions.java

```diff
@@ -1589,6 +1619,37 @@ public OpenTelemetry getOpenTelemetry() {
     }
   }
 
+  @Override
+  public ApiTracerFactory getApiTracerFactory() {
+    // Prefer any direct ApiTracerFactory that might have been set on the builder.
+    return MoreObjects.firstNonNull(super.getApiTracerFactory(), getDefaultApiTracerFactory());
+  }
+
+  private ApiTracerFactory getDefaultApiTracerFactory() {
```

Comment:
Probably out of scope for this PR, once you start using Gax metrics, you need to create a CompositeTracerFactory that includes both `OpenTelemetryApiTracerFactory` and [MetricsTracerFactory](https://github.com/googleapis/sdk-platform-java/blob/main/gax-java/gax/src/main/java/com/google/api/gax/tracing/MetricsTracerFactory.java).

Reply:
Thanks for the link. (And indeed for now out-of-scope, the immediate requirement is more RPC tracing, which will help customers understand why a given transaction takes X time.)